### ISSUE TITLE
Add language selector

### ISSUE THREAD

**Issue by furdiburd** [JC-Movies/main-website#1]
just a cross request from reddit.

It would be much more better if pepole speaking other language than english could use your site with selecting their preferred language and getting movies (if those exists for that one) with that one voice.

**Comment by JC-Movies** [comment]
Yeah I'm working on i18n and we'll require translators from community
